users/player-info-forwarding.rst

```diff
@@ -17,7 +17,22 @@ forwarding player information to your servers:
 Configuring modern forwarding
 -----------------------------
 
-Currently, no server implementations support Velocity's modern forwarding.
+Currently, only recent versions of Paper 1.13.1+ support Velocity's modern forwarding.
```

Comment:
Include build 377 or higher

Reply:
Should I put the information about which build supports it here, or under the `Paper` subheading? It's Paper-specific, so I don't think it necessarily belongs immediately below the `Configuring modern forwarding` heading.